src/main/java/uk/ac/ebi/embl/flatfile/writer/embl/FTWriter.java

```diff
@@ -30,33 +30,53 @@
 /** Flat file writer for the feature table.
  */
 public class FTWriter extends FlatFileWriter {
-	
-	public FTWriter(Entry entry, boolean sortFeatures, 
-			boolean sortQualifiers, WrapType wrapType) {
+
+	public FTWriter(Entry entry, boolean sortFeatures,
+					boolean sortQualifiers, WrapType wrapType) {
+		super(entry, wrapType);
+		this.sortFeatures = sortFeatures;
+		this.sortQualifiers = sortQualifiers;
+	}
+
+	public FTWriter(Entry entry, boolean sortFeatures,
+					boolean sortQualifiers, boolean excludeSource, WrapType wrapType) {
 		super(entry, wrapType);
 		this.sortFeatures = sortFeatures;
 		this.sortQualifiers = sortQualifiers;
+		this.excludeSource = excludeSource;
 	}
 
 	private boolean sortFeatures;
 	private boolean sortQualifiers;
+	private boolean excludeSource;
 
 	public boolean write(Writer writer) throws IOException {
 		Vector<Feature> features = new Vector<Feature>(entry.getFeatures());
 		if (features == null ||
 			features.size() == 0) {
 			return false;
 		}
+		if (excludeSource &&
```

Comment:
- It is possible to have more than one source feature in a flat file.
- However, entries that are being processed by our sequence processing pipelines have always exactly one source feature, so this code here should work.